### ISSUE TITLE
[Feature]: About commercial use of Masking Helper

### ISSUE THREAD

**Issue by ShotaYmasaki** [Gourieff/comfyui-reactor-node#364]
### Feature description

Thank you for the great code!

I understand that ReactorFaceSwap uses InsightFace, so it cannot be used commercially.
I don't think InsightFace is used in MaskingHelper.
In that case, is it correct to assume that commercial use is possible?

**Comment by Gourieff** [comment]
Yes, you can use it commercially, MaskingHelper uses models which go under Apache License 2.0 (SAM models) and under GNU Affero General Public License v3.0 (Ultralytics models)
So, there are no any limitations for Commercial use as far as I know

**Comment by ShotaYmasaki** [comment]
thank you for your reply!

So I would like to ask a question: Do you know of any models that can use FaceSwap, which is an alternative to Insightface?
